plugins/woocommerce/src/Blocks/BlockTypes/MiniCartItemsBlock.php

```diff
@@ -92,6 +94,9 @@ protected function render_experimental_iapi_markup( $attributes, $content, $bloc
 								<td class="wc-block-cart-item__product">
 									<div class="wc-block-cart-item__wrap">
 										<a data-wp-text="state.cartItemName" data-wp-bind--href="state.cartItem.permalink" class="wc-block-components-product-name"></a>
+										<div data-wp-bind--hidden="!state.isAvailableInBackorder" class="wc-block-components-product-badge wc-block-components-product-backorder-badge" hidden>
```

Comment:
I've added `hidden` to the HTML because it is the pattern I saw in the other elements. However, I wonder if we should initialize the state from the server somehow.

Reply:
Yeah, the `hidden` attribute should not be required there, as I pointed out in https://github.com/woocommerce/woocommerce/pull/59399#discussion_r2185676103. Also, for SSR, it would be good to initialize these values in the state.

I checked it, and SSR is not being applied for some reason here. 🤔